src/app/contact/contact-form/contact-form.component.ts

```diff
@@ -17,19 +23,229 @@ interface User {
 export class ContactFormComponent implements OnInit {
 	private ContactForm: FormGroup;
 	private submitted: boolean = false;
+	private selectedValue = null;
+	private countries:countrycode[] = [{code:213 , name:'Algeria (+213)'}, 
```

Comment:
Not here, too long source file add somewhere in shared, also try to find if some library is available for country codes.